pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -816,6 +795,44 @@ func (s *DiscoveryServer) removeEdsCon(clusterName string, node string) {
 	}
 }
 
+func (s *DiscoveryServer) updateEdsClusters(added sets.Set, removed sets.Set, connection *XdsConnection) {
```

Comment:
rename to updateEdsClients or updateEdsConnections?

Reply:
changed to `updateEdsClients`.